Differential gene expression analysis relies on several operations you have already used, such as examining the quality of a set of sequencing reads and trimming them if necessary, indexing a reference genome, and mapping reads. The analysis also requires you to access data from the GEO database, as you did for the Explore module. You'll then apply new skills: converting your mapped reads to a table showing the raw read counts for each gene across each sample, then using packages in RStudio to analyze the counts to find genes that are expressed at significantly different levels between controls and experimental samples.

The overall workflow is as follows:
![alt text](https://github.com/jamiehenzy/Genome_analysis_DGE/blob/assets/DGE_workflow.png)

                       <img src="https://github.com/jamiehenzy/Genome_analysis_DGE/blob/assets/DGE_workflow.png" alt="DGE_workflow" width="400" style="display: block; margin: 0 auto" />
